external/model-preparation-algorithm/tests/test_xai.py

```diff
@@ -9,7 +9,6 @@
 from mmdet.models import build_detector
 
 from mpa.modules.hooks.auxiliary_hooks import ReciproCAMHook, DetSaliencyMapHook
-from mpa.det.stage import DetectionStage
 

```

Comment:
For me this test did not pass without this import locally. @dongkwan-kim01 could you please double check?

Reply:
@dongkwan-kim01 Could you check this? I want to merge this branch

Reply:
@negvet @sungmanc Yes I'll check it. Sorry for late response

Reply:
You're right it's making error. I've added with flake8 exception